config/default.cfg

```diff
@@ -623,6 +623,13 @@ cfg$gms$cm_carbonprice_temperatureLimit <- 1.8 # def = 1.8; not-to-exceed temper
 
 # climate damages
```

Comment:
I think the newly introduced module realizations are missing in default.cfg

Reply:
fixed that.